test/karma.conf.js

```diff
@@ -30,14 +20,26 @@ module.exports = function(config) {
       terminal: false
     },
     customLaunchers: {
-      ChromeHeadlessNoSandbox: {
+      ChromeHeadlessDisableAutoplayPolicy: {
         base: 'ChromeHeadless',
-        flags: ['--no-sandbox', '--autoplay-policy=no-user-gesture-required']
-      },
-      ChromeCanaryHeadlessNoSandbox: {
-        base: 'ChromeCanaryHeadless',
-        flags: ['--no-sandbox', '--autoplay-policy=no-user-gesture-required'],
-       }
+        flags: ['--no-sandbox', '--no-user-gesture-required']
+      }
+    },
+    detectBrowsers: {
+      usePhantomJS: false,
+      postDetection: function(browsers) {
+        var newBrowsers = [];
+
+        if (browsers.indexOf('Chrome') !== -1) {
```

Comment:
Not sure if I'm the only one, but I was confused at first what this was supposed to do. Maybe a comment saying "use headless mode for detected chrome and firefox"?

Reply:
I'll add a comment communicating the intent. The idea is to check if you have Chrome installed, if so then we'll use the custom browser with all the flags